catena4460_bsec_ulp/catena4460_bsec_ulp.ino

```diff
@@ -44,37 +44,41 @@ Author:
 
 using namespace McciCatena;
 
-#define STATE_SAVE_PERIOD  UINT32_C(3 * 60 * 1000) // 3 minutes: basically any change.
+constexpr uint32_t STATE_SAVE_PERIOD = UINT32_C(3 * 60 * 1000); // 3 minutes: basically any change.
```

Comment:
You don't need UINT32_C() after you make it a `constexpr uint32_t`.

Reply:
Suggested changes have been done.